balancer/rls/internal/cache/cache.go

```diff
@@ -141,7 +141,7 @@ type LRU struct {
 // The cache package trusts the RLS policy (its only user) to supply a default
 // minimum non-zero maxSize, in the event that the ServiceConfig does not
 // provide a value for it.
-func NewLRU(maxSize int, onEvicted func(Key, *Entry)) *LRU {
+func NewLRU(maxSize int64, onEvicted func(Key, *Entry)) *LRU {
```

Comment:
Probably not in this PR.
Comment that `maxSize` is the memory size, not length?

Reply:
Ack.